CHANGES/438.feature

```diff
@@ -0,0 +1,2 @@
+Added `reset` command to access_policies.
+Changed `--permissions-assignment` to `--creation-hooks` to follow pulpcore 3.17 naming.
```

Comment:
pulp-cli uses a different docs building process that doesn't need double marks right?

Reply:
Also, do we not require issues be filed for new features? I realized that the changelog is the PR number itself.

Reply:
Yes, It's Markdown, not RST.